comanage_utils.py

```diff
@@ -81,20 +82,27 @@ def call_api2(method, target, endpoint, authstr, **kw):
 
 def call_api3(method, target, data, endpoint, authstr, **kw):
     req = mkrequest(method, target, data, endpoint, authstr, **kw)
-    trying = True
-    currentTimeout = MIN_TIMEOUT
-    while trying:
+    retries = 0
+    currentTimeout = TIMEOUT_MIN
+    requestingStart = time.time()
```

Comment:
```suggestion
    current_timeout = TIMEOUT_MIN
    requesting_start = time.time()
```

https://peps.python.org/pep-0008/#function-and-variable-names